concordium-node/src/configuration.rs

```diff
@@ -511,6 +510,14 @@ pub struct ConnectionConfig {
         env = "CONCORDIUM_NODE_CONNECTION_DEDUPLICATION_HASHING_ALGORITHM"
     )]
     pub deduplication_hashing_algorithm: DeduplicationHashAlgorithm,
+    #[structopt(
+        long = "max-normal-keep-alive",
+        help = "Max seconds to hold connection to dead \"normal\" node before discarding. Must be \
```

Comment:
```suggestion
        help = "Max seconds to keep alive an inactive connection to a \"normal\" node before discarding. Must be \
```

Reply:
You'll probably need to reformat so don't just apply this suggestion even if you agree with it. :)